designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -273,6 +276,10 @@ class DBProcessService(deploymentService: DeploymentService,
     )
   }
 
+  override def getProcessActions(id: ProcessId): Future[List[ProcessAction]] = {
```

Comment:
I wonder if this Service is starting to be a god service... Maybe it's time to provide ActionService?

Reply:
we discuss that it will be done in the future